tests/bgp/test_bgp_update_timer.py

```diff
@@ -58,10 +65,13 @@ def log_bgp_updates(duthost, iface, save_path, ns):
         duthost.asic_instance_from_namespace(ns).ns_arg,
         start_pcap,
     )
-    start_pcap = "nohup {}{} &".format(
+    start_pcap_cmd = "nohup {}{} &".format(
         duthost.asic_instance_from_namespace(ns).ns_arg, start_pcap
     )
-    duthost.shell(start_pcap)
+    duthost.shell(start_pcap_cmd)
+    if not wait_until(WAIT_TIMEOUT, 5, 20, lambda: is_tcpdump_running(duthost, start_pcap),):
```

Comment:
Did we see the case when `is_tcpdump_running` returns `False`? I think the process should be there immediately after we run the command `nohup tcpdump xxxx`.
If we suspect the issue was caused by tcpdump didn't start the capture before the bgp update was sending out, I suggest adding some sleep here.

Reply:
Agree with the suspect part of `is_tcpdump_running` may returns 'True` immediately.
But still, I like Jie's way of actively detecting if tcpdump is running because we can't decide a precise sleep time.

Reply:
My purpose here is
1. Check tcpdump process do started, sometime if memory is not enough or tcpdump tool options is not right, it will fail to run 
2. Make sure tcpdump process do scheduled by kernel,  or say ready start to capture packets. We can make sure "nohup tcpdump &" can work immediately as it's based on the nice value of user space process. Based on wait_until(). The initial delay is 20 seconds which is enough for tcpdump tool ready to capture packets. 

The issue in original test case is, it ran tcpdump without any waiting and trigger bgp update immediately, so on some environment bgp update send out but actually tcpdump not scheduled to capture packets yet.

I modified the code to make it clear for understanding. Please help have look at the latest diff.